include/kokkos-utils/tests/fixtures/ExecutionSpaceInstanceFixture.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef KOKKOS_UTILS_TESTS_HELPERS_HPP
+#define KOKKOS_UTILS_TESTS_HELPERS_HPP
+
+#include "kokkos-utils/concepts/ExecutionSpace.hpp"
+
+namespace Kokkos::utils::tests
+{
+
+//! Fixture class that creates an new execution space instance.
+template <Kokkos::utils::concepts::ExecutionSpace Exec>
+class ExecutionSpaceInstanceFixture : public virtual ::testing::Test
```

Comment:
We'll need to think a bit about how to deal with the dependency on `gtest` that we create here.

Reply:
It's not needed actually. Let's get rid of it.